data/helpers.d/filesystem

```diff
@@ -80,3 +80,31 @@ properly with chmod/chown." >&2
     chmod 755 $TMP_DIR
     echo $TMP_DIR
 }
+
+# Remove a file or a directory securely
+#
+# usage: ynh_secure_remove path_to_remove
+# | arg: path_to_remove - File or directory to remove
+ynh_secure_remove () {
+	path_to_remove=$1
```

Comment:
I am ok with forbidding * and it is fine to me to use the helper once for each file (or folder) to remove.
And we can still do a "for" loop if required to compact code a bit.